docs/developers/internals/handoffs.md

```
@@ -0,0 +1,518 @@
+# - Success criteria
+```
+
+### Configuration
+
+```yaml
+# config/workflow_config.yaml
+workflow:
+  handoff:
+    enabled: true
+    evaluation_threshold: 0.7
+    enable_handoffs: true # Enable structured handoffs
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)